waspc/cli/Wasp/Cli/Command/Db.hs

```diff
@@ -38,10 +38,12 @@ makeDbCommand cmd = do
 
   waspSaysC $ asWaspStartMessage "Setting up database..."
   chan <- liftIO newChan
-  -- NOTE(matija): What we do here is make sure that Prisma CLI is installed because db commands
-  -- (e.g. migrate) depend on it. We run setupServer which does even more than that, so we could make
-  -- this function more lightweight if needed.
-  (_, dbSetupResult) <- liftIO (concurrently (readJobMessagesAndPrintThemPrefixed chan) (setupServer genProjectDir chan))
+
+  -- NOTE(matija): What we do here is make sure that Prisma CLI is installed
+  -- because db commands (e.g. migrate) depend on it. We run
+  -- installNpmDependencies for the server which does even more than that, so
+  -- we could make this function more lightweight if needed.
+  (_, dbSetupResult) <- liftIO (concurrently (readJobMessagesAndPrintThemPrefixed chan) (installNpmDependencies genProjectDir chan))
```

Comment:
Do we need to do `installNpmDependencies` here anymore, now that `npm install` is part of generation? I _think_ the `npm install` will happen even without this because we compile before this.

Reply:
I think Shayne is right: makeDbCommand runs compile, which in the end calls out setup code. So this whole logic can be removed here as it should be guaranteed the setup phase has completed.

Reply:
Given that we execute `compile`, the whole logic after 'waspSaysC` including all the error handling can go away, because as far as I can see, this is only to deal with an `npm install` error, which cannot happen at this stage anymore. `compile` itself can fail but it throws a compilation failed error already.

So I think the correct response would be to remove everything after ` waspSaysC $ asWaspStartMessage "Setting up database..."` in this function. What do you think?

Reply:
Yeah it is tricky... it is almost like all we are left with there then is just the call to `compile`, so we may be able to just remove that whole function and call `compile` from `runDbCommand` directly in place of `makeDbCommand`?

Reply:
`makeDbCommand` was here to ensure that before any db command, correct setup is done. Even if that setup is only compiling the code, I would still leave as part of `makeDbCommand`.

Reply:
But we can remove everything else, yes.